### DISCUSSION TITLE
Hex Gradient oneblock -> generator

### DISCUSSION THREAD

**Discussion by xOpasik** [BG-Software-LLC/SuperiorSkyblock2#1183] [Pending Feature Requests] [answered]
![image](https://user-images.githubusercontent.com/105610066/172558203-b61853ec-824f-4238-9281-cb8837074676.png)
![image](https://user-images.githubusercontent.com/105610066/172558299-90bfaada-0043-4a42-96ed-db7a6f3d23e0.png)
Proposes to add the ability to create strings from gradients. I tried every gradient format from the screenshot and unfortunately none of them worked -> I did a restart every time

**Comment by OmerBenGera** [comment, marked as answer]
Next time check out the wiki page before suggesting already implemented features:
https://wiki.bg-software.com/superiorskyblock/overview/messages
